docs/compression_algorithms/CompressWeights.md

```diff
@@ -389,6 +395,90 @@ This modification applies only for patterns `MatMul-Multiply-MatMul` (for exampl
     </tr>
 </table>
 
+Here is the perplexity and accuracy with data-free and data-aware mixed-precision INT4-INT8 weight compression for different language models on the [lambada openai dataset](https://huggingface.co/datasets/EleutherAI/lambada_openai).
+`_scale` suffix refers to the data-aware mixed-precision with Scale Estimation algorithm.
+`r10` means that embeddings and lm_head have INT8 precision and all other linear layers have INT4 precision.
```

Comment:
r10 should be r100? fp should be fp16? I would also change the order to align with prev tables.

Reply:
Done